src/chaplin.coffee

```diff
@@ -1,23 +1,43 @@
 'use strict'
 
+import Application from './chaplin/application.coffee'
```

Comment:
Had to keep extensions, else rollup doesn't resolve the files. See rollup/rollup#1053